src/components/ProfilePage/DateJoined.jsx

```diff
@@ -7,7 +7,7 @@ function DateJoined({ date }) {
 
   return (
     <FormattedMessage
-      id="profile.date.joined"
+      id="profile.datejoined.member.since"
```

Comment:
@abutterworth @albemarle: When and if code gets refactored into different components, we should make it clear (process-wise) if we would **not** want the keys to also be refactored so that translators don't have to retranslate the same messages.  If that is the case, only the "profile" part of the namespace will probably be reliable longterm.

Reply:
Good point!